src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -1,33 +1,91 @@
 package org.jetbrains.research.transformations.util
 
+import org.jetbrains.research.transformations.util.ParserSetup.checkSetup
 import java.io.File
+import java.io.InputStream
+import java.net.URL
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
 import java.util.logging.Logger
+import net.lingala.zip4j.ZipFile
+import org.apache.commons.lang3.SystemUtils
+import org.jetbrains.research.transformations.util.Util.getTmpPath
+import org.jetbrains.research.transformations.util.Util.runProcessBuilder
+
 
 /**
  * [ParserSetup] class is created for parser setup before using Gumtree with python code.
  * Also [checkSetup] should be called before running tests.
  */
 object ParserSetup {
     private val LOG = Logger.getLogger(javaClass.name)
+
+    private const val PYTHON3_PROPERTY = "ac.p3.path"
+    private const val PYTHONPARSER_PROPERTY = "gt.pp.path"
+
+    private const val PARSER_REPOSITORY_ZIP_URL =
+        "https://github.com/JetBrains-Research/pythonparser/archive/master.zip"
+    private const val PARSER_ZIP_NAME = "master.zip"
+
+    // Relative path in the parser repository
+    private const val REPOSITORY_ROOT_FOLDER = "pythonparser-master"
+    private const val PARSER_RELATIVE_PATH = "$REPOSITORY_ROOT_FOLDER/src/main/python/pythonparser/pythonparser_3.py"
+    private const val INVERSE_PARSER_RELATIVE_PATH =
+        "$REPOSITORY_ROOT_FOLDER/src/main/python/inverse_parser/inverse_parser_3.py"
+
     private const val PARSER_NAME = "pythonparser"
-    private val TARGET_PATH = "${getTmpPath()}/$PARSER_NAME"
+
+    private val TARGET_PARSER_PATH = "${getTmpPath()}/$PARSER_NAME"
+    private val TARGET_INVERSE_PARSER_PATH = "${getParserRepositoryPath()}/$INVERSE_PARSER_RELATIVE_PATH"
```

Comment:
let's make them consts

Reply:
Sorry, but I can not, because I use `getTmpPath` and `getParserRepositoryPath` functions

Reply:
Oh, sorry, didn't notice it. So let's don't use CAPS since it's confusing :)